.github/workflows/health_check.yml

```diff
@@ -51,3 +52,10 @@ jobs:
           git push --force origin gh-pages
         env:
           DOCSQA_ADMIN_TOKEN: ${{ secrets.DOCSQA_ADMIN_TOKEN }}
+      - if: steps.run-health-check.outputs.HEALTH_CHECK_FAILED == 'true'
+        uses: 8398a7/action-slack@v3
+        with:
+          status: failure
+          text: |
+            :no_entry: DocsQA services health check failed
```

Comment:
Would you please add a link here to `jina.ai/docsQA-status`?

Reply:
Addressed.